packaging/crio.conf.d/microshift_amd64.conf

```diff
@@ -2,8 +2,19 @@
 selinux = true
 conmon = ""
 conmon_cgroup = "pod"
+default_env = [
+    "NSS_SDB_USE_CACHE=no",
+]
 cgroup_manager = "systemd"
 default_runtime = "crun"
+default_sysctls = [
+    "net.ipv4.ping_group_range=0 2147483647",
+]
+manage_ns_lifecycle = true
+absent_mount_sources_to_reject = [
+    "/etc/hostname",
+]
+drop_infra_ctr = true
```

Comment:
Can we add a comment in the file on why these settings are added? There's nothing in the description of the PR.

Reply:
I added the comment in the commit message.